src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/RuntimeExports.cs

```diff
@@ -116,7 +116,7 @@ public static unsafe object RhBox(MethodTable* pEEType, ref byte data)
             {
                 fixed (byte* pFields = &result.GetRawData())
                 fixed (byte* pData = &dataAdjustedForNullable)
-                    InternalCalls.memmove(pFields, pData, pEEType->ValueTypeSize);
+                    Unsafe.CopyBlock(pFields, pData, pEEType->ValueTypeSize);
```

Comment:
This can call CopyBlock that takes ref byte and we can delete the pinning.

Reply:
Done.